### PR TITLE
chore: pull in latest subxt changes

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: runtime/src/lib.rs [hunk 1]
```diff
@@ -60,21 +59,21 @@ pub const STABLE_PARACHAIN_CONFIRMATIONS: &str = "StableParachainConfirmations";
...
-        generated_type_derives = "Debug, Eq, PartialEq, Ord, PartialOrd, Clone"
+        generated_type_derives = "Eq, PartialEq, Ord, PartialOrd, Clone"
     )
 )]
 #[cfg_attr(
     feature = "standalone-metadata",
     subxt(
         runtime_metadata_path = "metadata-standalone.scale",
-        generated_type_derives = "Debug, Eq, PartialEq, Ord, PartialOrd, Clone"
+        generated_type_derives = "Eq, PartialEq, Ord, PartialOrd, Clone"
```
Comment (sander2) [inline runtime/src/lib.rs:76]: Why not derive debug? That can be quite useful in debugging

File: runtime/src/rpc.rs [hunk 2]
```diff
@@ -9,50 +11,24 @@ use crate::{BTC_RELAY_MODULE, STABLE_BITCOIN_CONFIRMATIONS, STABLE_PARACHAIN_CON
...
-        const DEFAULT_RUNTIME_VERSION: RuntimeVersion = RuntimeVersion {
-            spec_name: create_runtime_str!("testnet-parachain"),
-            impl_name: create_runtime_str!("testnet-parachain"),
-            authoring_version: 1,
-            spec_version: 0,
-            impl_version: 1,
-            transaction_version: 0,
-            apis: Cow::Owned(vec![]),
-        };
+        const DEFAULT_SPEC_VERSION: u32 = 0;
```
Comment (sander2) [inline runtime/src/rpc.rs:31]: Why move away from the runtime compatibility check?

File: runtime/src/types.rs [hunk 3]
```diff
@@ -271,3 +268,69 @@ mod h256_le {
         }
     }
 }
+
+mod dispatch_error {
```
Comment (sander2) [inline runtime/src/types.rs:272]: Uf this is quite ugly. Did you open an issue on subxt to see if it's possible to allow a more elegant solution? Also, how robust is this module? Will we get a compilation error if the dispatcherror type ever changes?

File: runtime/src/rpc.rs (hunk 2 above)
Comment (gregdhill) [reply runtime/src/rpc.rs:31]: `subxt` has redefined [RuntimeVersion](https://github.com/paritytech/subxt/blob/a7c54f74127b58dd0886b9f42b1f7318dfbc51f4/subxt/src/rpc.rs#L177-L198).

File: runtime/src/lib.rs (hunk 1 above)
Comment (gregdhill) [reply runtime/src/lib.rs:76]: Already added by subxt: https://github.com/paritytech/subxt/blob/a7c54f74127b58dd0886b9f42b1f7318dfbc51f4/codegen/src/types/derives.rs#L53

File: runtime/src/types.rs (hunk 3 above)
Comment (gregdhill) [reply runtime/src/types.rs:272]: https://github.com/paritytech/subxt/issues/426#issuecomment-1028917908

**Review by sander2 — APPROVED** [review]

**Pull request merged**